lib/rack/instrumentation/tracer.rb

```diff
@@ -30,6 +30,7 @@ def trace_request(env, scope)
 
         app.call(env).tap do |status_code, _headers, _body|
           span.set_tag('http.status_code', status_code)
+          span.set_tag('error', true) if status_code >= 500 && status_code < 600
```

Comment:
I think something like `span.set_tag('error', true) if (500..599).include?(status_code)` is maybe a bit more idiomatic.